ci/inspect_changes.sh

```diff
@@ -61,6 +61,39 @@ declare -A project_dirs=(
   [cccl_c_parallel]="c/parallel"
 )
 
+# Changes to files / directories listed here are ignored when checking if the
+# CCCL Infrastructure has been modified.
+# These are checked as regexes that match the beginning of the file path.
+ignore_paths=(
+  ".clang-format"
+  ".clangd"
+  ".devcontainer/README.md"
+  ".devcontainer/img"
+  ".git-blame-ignore-revs"
+  ".github/actions/docs-build"
+  ".github/ISSUE_TEMPLATE"
+  ".github/CODEOWNERS"
+  ".github/copy-pr-bot.yaml"
+  ".github/PULL_REQUEST_TEMPLATE.md"
+  ".github/workflows/backport-prs.yml"
+  ".github/workflows/build-docs.yml"
+  ".github/workflows/build-rapids.yml"
+  ".github/workflows/project_automation" # All project automation workflows
+  ".github/workflows/triage_rotation.yml"
+  ".github/workflows/update_branch_version.yml"
+  ".github/workflows/verify-devcontainers.yml"
+  ".gitignore"
+  ".pre-commit-config.yaml"
```

Comment:
I am not sure about this one. That could change the formatter which has consequences